package.json

```diff
@@ -85,6 +85,7 @@
     "process": "^0.11.10",
     "rdf-ext": "^1.3.5",
     "react": "^16.14.0",
+    "react-bootstrap-icons": "^1.5.0",
```

Comment:
new bootstrap-icon module since we are using a bootstrap icon for creating resources

Reply:
Using two different icon libraries seems like unnecessary bloat.  I think we should pick one icon set and use it.  I'd like to hear what @justinlittman's thought on this are.